ghw/libghw.c

```diff
@@ -555,7 +555,7 @@ ghw_read_array_subtype (struct ghw_handler *h, union ghw_type *base)
     }
   else
     {
-      /* Read bounds for the elements.  */
+      /* Read bounds for the elements.     */
```

Comment:
So what is the difference ?

Reply:
None. I don't know why `indent ghw/libghw.c` adds those.

Reply:
Then I would not keep those changes.  They really look random.

Reply:
Done.